static/app/views/performance/transactionSummary/transactionOverview/relatedIssues.tsx

```diff
@@ -31,7 +31,7 @@ type Props = {
 
 class RelatedIssues extends Component<Props> {
   getIssuesEndpoint() {
```

Comment:
Does it still make sense to name it this?

Reply:
Maybe `getIssuesEndpointQueryParams`?